scripts/release/constants.ts

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+
+enum PACKAGE_TYPES {
+    NotPackage = 0,
+    Wrapper,
+    IntegrationWrapper,
+    Package
+}
+
+enum PACKAGE_SUBTYPE {
+    INDEPENDENT = 0,
+    API,
+    COMPOSABLE,
+    THEME,
+    CLI
+}
+
+enum RELEASE_GRADATIONS {
+    path = 1,
```

Comment:
please next time try to avoid using same names as the imported packages

Reply:
I will